docs/general/administration/hardware-acceleration/rockchip.md

```diff
@@ -229,6 +235,74 @@ Root permission is required.
 
 3. Enable RKMPP in Jellyfin and uncheck the unsupported codecs.
 
+#### Non-official LXC (Linux Containers)
+
+This setup might be useful for those, who use RK3588/3588S SoC as [Proxmox](https://www.proxmox.com/en/) host, where LXC is the official and the only supported way of doing lightweight virtualiztion with the help of system containers (LXC) vs application containers (docker). As of today Proxmox team does not support ARM platforms as hosts - and probably will never do - however successful deployments on ARM devices [are possible](https://github.com/jiangcuo/Proxmox-Port?tab=readme-ov-file).
+
+LXC setup idea is a bit similar to docker - you need to pass the **device files** of VPU from host to LXC and enable the **privileged mode** (see also important "_note_" below about privileged LXC containers).
+1. To find the list of device files to pass inside container, use the next one-liner in Linux host:
+
+```
+root@pve:~# for dev in dri dma_heap mali0 rga mpp_service iep mpp-service vpu_service vpu-service hevc_service hevc-service rkvdec rkvenc vepu h265e ; do [ -e "/dev/$dev" ] && echo "device /dev/$dev";  done
+device /dev/dri
+device /dev/dma_heap
+device /dev/mali0
+device /dev/rga
+device /dev/mpp_service
+```
+
+Example of the minumum requried extra (not full) container configuration to make VPU hardware accelearion working is presented below:
+
+```
+lxc.mount.entry: /dev/dri dev/dri none bind,optional,create=dir
+lxc.mount.entry: /dev/dma_heap dev/dma_heap none bind,optional,create=dir
+lxc.mount.entry: /dev/mpp_service dev/mpp_service none bind,optional,create=file
+lxc.mount.entry: /dev/rga dev/rga none bind,optional,create=file
+lxc.mount.entry: /dev/mali0 dev/mali0 none bind,optional,create=file
+lxc.cgroup2.devices.allow: a
+```
+
+Use [lxc.container.conf](https://linuxcontainers.org/lxc/manpages//man5/lxc.container.conf.5.html) man page for reference, what _lxc.mount.entry_ and _lxc.cgroup2.devices_ options are used for.
+If you want - and you have concerns about it - you can replace _lxc.cgroup2.devices.allow: a_ rule with more restricted "_allowlist device program_" (example could be found inside the same man page) enumerating each character VPU device, that container namespace should have access inside the host.
+
+:::note
+
+Privileged LXC containers are considered unsafe by design - read more [here](https://linuxcontainers.org/lxc/security/). This guide however does not cover steps required to make jellyfin VPU hardware acceleration working in unprivileged container.
+
+:::
+
+2. Install [jellyfin package](https://jellyfin.org/docs/general/installation/linux) for your Linux flavor into LXC container or optionally you can even use an official docker image inside LXC container.
+3. Verify OpenCL runtime status as following - example is collected from LXC runtime of Ubuntu Jammy - steps are literally the same as in case of docker deployment:
```

Comment:
```suggestion
3. Verify OpenCL runtime status as following - example is collected from LXC runtime of Ubuntu Jammy - steps are the same as docker deployments:
```
Are the steps of the verify on linux section insufficient?

Reply:
In fact Mali GPU user-space drivers should be installed only in the runtime, where OpenCL-based HDR tone-mapping is required - it could be host or docker or LXC - depending where you try to enable/configure RKMPP hardware acceleration.

For example I completely removed **libmali** package from the host and still LXC container can access OpenCL runtime via /dev/mali0 device. Opposite is not True - if I remove **libmali** package from LXC container and install it into Debian host, access to OpenCL runtime from container does not work, however it works as it should from the host itself.

I see in this tutorial information, that ARM Mali OpenCL runtime (libmali) should be installed on host in case of docker virtualization (step 5) - I can neither agree or disagree with that - however for LXC case this step is not required as soon as you don't plan using RKMPP HWA directly in the host.